employee_manual/travelling.md

```diff
@@ -0,0 +1,29 @@
+# Travelling
+
+## Commuting to your main office
+Commuting does not count as work time.
+
+However, if your commute distance is long enough you might be eligible for "Deduction for transport to work" (Befordringsfradrag). More information is available at [skat.dk](https://skat.dk) (use the search feature).
```

Comment:
I don't think this bit of information belongs in our policy. It concerns the individual and I would expect (the few people it concerns) to know themself. So with the purpose of keeping our policies as short and concise as possible, I would vote for removing it.

Reply:
I tried to make this part require as little maintenance as possible, hence the "go to website and search".

What about the "Commuting does not count as work time" part? I think that without it some people might misunderstand some of the other parts of the document.

Reply:
Agreed! That bit is worth keeping. Otherwise we open up to creative interpretations 😄

Reply:
I would leave the commuting part, it was only the stuff in line 6-7 I voted for removing (might need a comment in Github course).